### ISSUE TITLE
it give me this error: Unable to find cookie from Roblox Studio registry - dm me SumTony#6283

### ISSUE THREAD

**Issue by tonyballony** [NightfallGT/Mercurial-Grabber#30]
i need help w that

**Comment by pancak3boy** [comment]
its because you have to be logged in to roblox studio to get the cookie

**Comment by tonyballony** [comment]
> its because you have to be logged in to roblox studio to get the cookie

so i need to be in roblox studio when they open file?

**Comment by NBDMatr1x** [comment]
> > its because you have to be logged in to roblox studio to get the cookie
> 
> so i need to be in roblox studio when they open file?

no, the reason u get no cookies is cos the person u are grabbing isnt logged in.

**Comment by pancak3boy** [comment]
yas
